src/main/java/uk/gov/hmcts/divorce/citizen/notification/ApplicationSubmittedNotification.java

```diff
@@ -49,16 +55,59 @@ public void sendToApplicant2(final CaseData caseData, final Long id) {
             notificationService.sendEmail(
                 caseData.getApplicant2EmailAddress(),
                 JOINT_APPLICATION_SUBMITTED,
-                templateVars(caseData, id, caseData.getApplicant2(), caseData.getApplicant1()),
+                citizenTemplateVars(caseData, id, caseData.getApplicant2(), caseData.getApplicant1()),
+                caseData.getApplicant1().getLanguagePreference()
+            );
+        }
+    }
+
+    @Override
+    public void sendToApplicant1Solicitor(final CaseData caseData, final Long caseId) {
+
+        String solicitorEmail = caseData.getApplicant1().getSolicitor().getEmail();
+        if (!caseData.getApplicationType().isSole() && solicitorEmail != null && !solicitorEmail.isBlank()) {
+            log.info("Sending joint application submitted notification to applicant 1 solicitor for case : {}", caseId);
+
+            notificationService.sendEmail(
+                solicitorEmail,
+                SOLICITOR_JOINT_APPLICATION_SUBMITTED,
+                solicitorTemplateVars(caseData, caseId, caseData.getApplicant1()),
                 caseData.getApplicant1().getLanguagePreference()
             );
         }
     }
 
-    private Map<String, String> templateVars(CaseData caseData, Long id, Applicant applicant, Applicant partner) {
+    @Override
+    public void sendToApplicant2Solicitor(final CaseData caseData, final Long caseId) {
+
+        String solicitorEmail = caseData.getApplicant2().getSolicitor().getEmail();
+        if (!caseData.getApplicationType().isSole() && solicitorEmail != null && !solicitorEmail.isBlank()) {
```

Comment:
you can use `org.apache.commons.lang3.StringUtils.isNotEmpty()` instead of two checks

Reply:
implemented this, thanks for the suggestion